interface/forms/misc_billing_options/date_qualifier_options.php

```diff
@@ -43,18 +34,16 @@ function genProviderSelect($selname, $toptext, $default=0, $disabled=false) {
     "AND active = 1 AND ( info IS NULL OR info NOT LIKE '%Inactive%' ) " .
     "ORDER BY lname, fname";
   $res = sqlStatement($query);
-  echo "   <select name='" . attr($selname) . "'";
+  echo "<select name='" . attr($selname) . "'>";
```

Comment:
above needs to be reverted since the below `disabled` needs to be within the html select tag.